defs.mak

```diff
@@ -365,3 +365,11 @@ RUNTEST_COMMAND=$(TOP)/scripts/runtest
 ##==============================================================================
 
 MKROOTFS=$(TOP)/scripts/mkrootfs
+
+##==============================================================================
+##
+## SQUASH-APPDIR script
+##
+##==============================================================================
+
+SQUASH-APPDIR=$(TOP)/scripts/squash-appdir.sh
```

Comment:
I think "slim" is a better term than "squash". :-)

Reply:
I guess it is a matter of taste. I like "squash" better than "slim" in this context.

Reply:
Would "clean" be better? When the context was "squash" and without seeing the script, the first thing I thought was we are taking all the appdirs and combining them to reduce the build time for the rootfs by building one instead of N, but seeing the script I realize we are reducing the size of individual appdirs. Just my first impression here.

Reply:
Hmm? Clean makes me think it does away with the appdir althogether.

Reply:
Smash? Crush? Deflate?

Reply:
How about prune?